### DISCUSSION TITLE
I hope the ID/relay service can support WSS

### DISCUSSION THREAD

**Discussion by summanzhao** [rustdesk/rustdesk-server#352] [General]
**Is your feature request related to a problem? Please describe.**
At present, most browsers will directly locate the default opening address as https, and the https address cannot access the WebSocket service of ws by default, so implementing WSS feels very necessary.

**Describe the solution you'd like**
I hope the ID/relay service can support WSS.

**Comment by TBG-FR** [comment]
I've been stuck for months with that, having to keep a HTTP-only web client, which some browsers or phones wouldn't access because of their settings (always default/force HTTPS to be secure)

Finally made it, if that can help, here you go 😉 https://github.com/rustdesk/rustdesk/discussions/6023#discussioncomment-8687562
